manager/orchestrator/restart/restart.go

```diff
@@ -101,6 +104,7 @@ func (r *Supervisor) waitRestart(ctx context.Context, oldDelay *delayedStart, cl
 // Restart initiates a new task to replace t if appropriate under the service's
 // restart policy.
 func (r *Supervisor) Restart(ctx context.Context, tx store.Tx, cluster *api.Cluster, service *api.Service, t api.Task) error {
+
```

Comment:
tiny nit: remove this bonus line. not a blocker to getting this merged, if you forget, but while you're cleaning up other comments just nuke this.